pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -109,6 +127,49 @@ func TestInt_SecurityIntegrations(t *testing.T) {
 		assert.Equal(t, "SECURITY", si.Category)
 	}
 
+	assertFieldContainsList := func(details []sdk.SecurityIntegrationProperty, field, value string) {
```

Comment:
I think that:
1. `separator` should be one of the inputs
2. This method could be extracted as a helper (not necessarily now but in the future, so at least some TODO with a  new ticket's number) - and of course it should be then tested

Reply:
Fixed and added a TODO.